src/core/Header.tsx

```diff
@@ -135,6 +138,7 @@ const Header: React.FC<HeaderProps> = ({
   const [showMenu, setShowMenu] = useState(false);
   const [fadingOut, setFadingOut] = useState(false);
   const [scrollpointClasses, setScrollpointClasses] = useState<string>("");
+  const [bannerVisible, setBannerVisible] = useState(true);
```

Comment:
Will the banner always be visible? What happens if we don't have a notice/announcement to show?

Reply:
Great question, the banner is controlled from the website admin so it can be switched on/off at anytime

Reply:
@jamiehenson yes if the notice is null it won't be displayed. Notice object is passed from Voltaire/website which is API request from  https://ably.com/api/notice?url=/